Panic: slice bounds out of range in viewport when displaying issues

## Description

The `auto-worktree` command crashes with a panic when displaying the GitHub issues selection menu.

## Error

```
panic: runtime error: slice bounds out of range [18:11]
```

## Stack Trace

```
goroutine 1 [running]:
runtime/debug.Stack()
    runtime/debug/stack.go:26 +0x64
runtime/debug.PrintStack()
    runtime/debug/stack.go:18 +0x1c
github.com/charmbracelet/bubbletea.(*Program).recoverFromPanic(0x14000448000, {0x10333d520, 0x14000790240})
    github.com/charmbracelet/bubbletea@v1.3.6/tea.go:810 +0x90
github.com/charmbracelet/bubbletea.(*Program).Run.func2()
    github.com/charmbracelet/bubbletea@v1.3.6/tea.go:601 +0xc4
panic({0x10333d520?, 0x14000790240?})
    runtime/panic.go:783 +0x120
github.com/charmbracelet/bubbles/viewport.Model.visibleLines({0x88, 0xfffffffffffffff9, {{{0x140001bc7e0, 0x3, 0x3}, {{...}, {...}}, 0x0}, {{0x14000765ec0, 0x2, ...}, ...}, ...}, ...})
    github.com/charmbracelet/bubbles@v0.21.0/viewport/viewport.go:150 +0x204
github.com/charmbracelet/bubbles/viewport.(*Model).GotoBottom(0x14000232008)
    github.com/charmbracelet/bubbles@v0.21.0/viewport/viewport.go:348 +0xd4
github.com/charmbracelet/bubbles/viewport.(*Model).SetContent(0x14000232008, {0x14000723200?, 0x0?})
    github.com/charmbracelet/bubbles@v0.21.0/viewport/viewport.go:131 +0xb4
github.com/charmbracelet/gum/filter.model.View({{{0x0, 0x0}, {0x140005df2cc, 0x2}, {0x14000807b48, 0x18}, 0x0, 0x2a, {0x1f972880, {0x0, ...}, ...}, ...}, ...})
    github.com/charmbracelet/gum/filter/filter.go:233 +0xabc
github.com/charmbracelet/bubbletea.(*Program).eventLoop(0x14000448000, {0x1033733d0?, 0x14000766008?}, 0x1400066c230)
    github.com/charmbracelet/bubbletea@v1.3.6/tea.go:533 +0x684
github.com/charmbracelet/bubbletea.(*Program).Run(0x14000448000)
    github.com/charmbracelet/bubbletea@v1.3.6/tea.go:679 +0x840
github.com/charmbracelet/gum/filter.Options.Run({{0x14000446820, 0xa, 0xa}, {0x140005defd0, 0x3}, {{0x140005defe0, 0x3}, {0x0, 0x0}, {0x140005deff8, ...}, ...}, ...})
    github.com/charmbracelet/gum/filter/command.go:146 +0x1034
```

## Root Cause

The panic occurs in `github.com/charmbracelet/bubbles/viewport.Model.visibleLines` at `viewport/viewport.go:150`. The error indicates an invalid slice operation where the start index (18) is greater than the end index (11), suggesting the viewport height calculation is producing a negative or invalid value.

## Steps to Reproduce

1. Run `auto-worktree` command
2. The issues list displays briefly
3. Application crashes with the panic

## Environment

- Platform: darwin
- OS Version: Darwin 24.6.0
- Date: 2025-12-31

## Additional Context

The program successfully fetched and started displaying the issues list before crashing:
- Issues #10 through #4 were visible in the output
- The crash appears to happen during the viewport rendering in the gum filter component

--- a/aw.sh
+++ b/aw.sh
@@ -754,7 +754,7 @@ _aw_issue() {
       return 1
     fi
 
-    local selection=$(echo "$issues" | gum filter --height 20 --placeholder "Type to filter issues...")
+    local selection=$(echo "$issues" | gum filter --placeholder "Type to filter issues...")
 
     if [[ -z "$selection" ]]; then
       gum style --foreground 3 "Cancelled"
@@ -837,7 +837,7 @@ _aw_pr() {
       return 1
     fi
 
-    local selection=$(echo "$prs" | gum filter --height 20 --placeholder "Type to filter PRs... (✓=passing ✗=failing ○=pending)")
+    local selection=$(echo "$prs" | gum filter --placeholder "Type to filter PRs... (✓=passing ✗=failing ○=pending)")
 
     if [[ -z "$selection" ]]; then
       gum style --foreground 3 "Cancelled"

